install_config/redeploying_certificates.adoc

```diff
@@ -437,6 +437,25 @@ by deleting existing secrets that contain service serving certificates or removi
 annotations to appropriate services.
 ====
 
+[[redeploying-named-certificates]]
+=== Redeploying Only Named Certificates
+
+The *_openshift-master/redeploy-named-certificates.yml_* playbook redeploys only named certificates. Running this playbook also completes serial restarts of master services.
+
+To redeploy named certificates only, change to the directory that contains the playbooks, and run this playbook.
+
+----
+$ cd /usr/share/ansible/openshift-ansible
+$ ansible-playbook -i <inventory_file> \
+    playbooks/openshift-master/redeploy-named-certificates.yml
+----
+
+[NOTE]
+====
+The *_openshift_named_certificates_* inventory file must contain certificates with the same name as in the *_master-config.yaml_* file. If the names of `certfile` and `keyfile` are changed, you must update the xref:../install_config/certificate_customization.adoc#configuring-custom-certificates[named certificate parameters] in the *_master-config.yaml_* file on each master node and restart the `api` and `controllers` services. The `cafile` with the full ca chain is added to `/etc/origin/master/ca-bundle.crt`.
```

Comment:
Not quite sure what does this mean:
`The *_openshift_named_certificates_* inventory file must contain certificates with the same name as in the *_master-config.yaml_* file`
Maybe
`The *_ openshift_master_named_certificates_* parameter in ansible inventory file must contain certificates with the same name as in the *_master-config.yaml_* file` ?

Reply:
@gpei sure, you want me to change it?

Reply:
@vwalek, please do. 👍

Reply:
on it